tests/README.md

```diff
@@ -0,0 +1,114 @@
+|    |    |____rosacli                            ---- ROSA CLI specific services/commands
+|    |____log                                     ---- tests logger
+|       
+|____prow_ci.sh
+```
+
+## Contibute to ROSA CLI tests
+
+Please read the structure and contribute code to the correct place
+
+<!-- ### Contribute to day1 case
+
+* Define profile in ./tests/ci/data/profiles folder
+* Define prepare steps according to the profile in rosa/tests/ci/profile_handler.go
+
+#### Day1 cases only do creation step, we have to define a day1 case to verify the day1 works well
+
+* Create the case in rosa/tests/e2e/<feature name>_test.go
+* Label the case with ***CI.Day1Post***
+* Label the case with importance ***CI.Critical***
+* Don't need to run creation step, just in BeforeEach step call function  ***PrepareRHCSClusterByProfileENV()*** it will load the clusterID prepared
+* Code for checking steps  only in the case -->
+
+### Contribute to day2
+
+* Create the case in rosa/tests/e2e/<feature name>_test.go
+* Label the case with ***CI.Day2***
+* Label the case with importance ***CI.Critical*** or ***CI.High***
+* Don't need to run creation step, just in BeforeEach step call function  ***PrepareRHCSClusterByProfileENV()*** it will load the clusterID prepared
+* Code for day2 actions and check step
+* Every case need to recover the cluster after the case run finished unless it's un-recoverable
+
+### Contribute to day3
+
+* day3 cases means the cases is destructive and cannot recover anymore. The actions will affect other cases like default worker pool deletion, it will affect a lot of day1 post verification
+* Create the case in rosa/tests/e2e/<feature name>_test.go
+* Label the case with ***CI.Day3***
+* Label the case with importance ***CI.Critical*** or ***CI.High***
+* Don't need to run creation step, just in BeforeEach step call function  ***PrepareRHCSClusterByProfileENV()*** it will load the clusterID prepared
+* Code for day3 actions and check step
+* Every case need to recover the cluster after the case run finished unless it's un-recoverable
+
+### Labels
+
+* Label your case with the ***CI.Feature<feature name>*** defined in rosa/tests/ci/labels/features.go
+* Label your case with importance defined in rosa/tests/ci/labels/importance.go
+* Label your case with ***CI.Day1Post/CI.Day2/CI.Day3*** defined in rosa/tests/ci/labels/runtime.go, according to the case runtime
+* Label your case with ***CI.Exclude*** if it fails CI all  the time and you can't fix it in time
+* Label you case with ***CI.NonClassicCluster/CI.NonHCPCluster*** if it does not fit a type of cluster
+
+## Running
+
+The cluster created by the automation scripts are shared across all of the test cases. [Why we need do this?](./docs/challenge.md).
+
+### Prerequisite
+
+Please read repo's [README.md](../README.md)
+For the test cases, we need `make install` to make the terraform provider installed to local
```

Comment:
No need `make install`. And this not for terraform provider

Reply:
updated. and still we need the rosa command to be in PATH or ?